source/app/views/student_dashboard/_navbar.html.erb

```diff
@@ -23,6 +32,10 @@
       <i class="fa-solid fa-comment"></i> Feedback
     <% end %>
 
+    <%= link_to settings_student_dashboard_index_path(course_id: @selected_course.id), class: "nav-signedin-link #{'active' if current_page?(settings_instructor_dashboard_index_path(course_id: @selected_course.id))}" do %>
```

Comment:
make sure its active if  the current page `settings_student_dashboard_index_path` and not `settings_instructor_dashboard_index_path`